website/docs/d/online_archive.html.markdown

```diff
@@ -43,6 +48,10 @@ data "mongodbatlas_online_archive" "test" {
 * `expire_after_days` - Number of days after the value in the criteria.dateField when MongoDB Cloud archives data in the specified cluster. Set this parameter when `type` is `DATE`.
 * `query` - JSON query to use to select documents for archiving. Atlas uses the specified query with the db.collection.find(query) command. The empty document {} to return all documents is not supported. Set this parameter when `type` is `CUSTOM`.
 
+### Data Expiration Rule
+* `expire_after_days` - Number of days used in the date criteria for nominating documents for deletion. Value must be between 7 and 9215.
```

Comment:
Content copied for Atlas API docs. Reached out to Atlas Extended II team to verify if this is correct. Inspired from the Atlas UI field input, a more representative description would be: `Number of days data is stored in  the Online Archive before it gets deleted.`